packages/dashboard/src/layout/LayoutUtils.ts

```diff
@@ -468,7 +468,7 @@ class LayoutUtils {
     replaceConfig?: Partial<ItemConfigType>;
     createNewStack?: boolean;
     focusElement?: string;
-    dragEvent?: DragSourceEvent;
+    dragEvent?: DragEvent;
```

Comment:
W/ the change to `types` in tsconfig, we could just use `React.DragEvent` here without adding an import. Not sure how you feel about that

Reply:
Same difference to me